conduct/COC_POLICY.md

```diff
@@ -0,0 +1,107 @@
+
+The current composition of the CoC Team is available in the [code of conduct][CoC] itself along with means to contact by email the whole CoC Team and each of its members individually.
+
+The CoC Team is responsible for maintaining and applying the [incident management processes][].
+
+The CoC Team is empowered to involve external people to resolve an issue provided these abide by the code of conduct and by this policy.
+
+### Delegation to projects
+
+Projects may opt-in to enforce the code of conduct themselves provided they meet the following requirements:
+
+1. Agree to abide by this policy.
+2. Provide a general email address that incidents can be reported to and document it alongside the code of conduct.
+3. Define and publicly document a process by which the project selects people responsible for managing incident reports. For Impact and At Large projects, at least two people must be selected.
+4. List the people responsible for managing incident reports alongside the code of conduct as outlined in the [Reporting incidents section](#reporting-incidents).
+5. Publicly document their decision-making process for responding to incident reports.
+6. Document the foundation's escalation and appeal process alongside the code of conduct.
+7. Participate in good faith in the escalation and appeal processes.
+
+Reporters, victims, and offenders may [escalate](#escalation) an issue to the CoC Team at any point in the process in case of a conflict of interest. They may [appeal](#appeals) the decision to the CoC Team up to thirty (30) days after a response was provided if they believe that the process was not followed properly.
+
+## Moderation
+
+Each project is free to implement moderation processes appropriate for the size and scope of the project provided these processes abide by the code of conduct and by this policy.
+
+## Escalation
+
+The CoC Team is empowered to manage escalation requests of incident reports managed by projects that have opted-in to enforce the code of conduct themselves.
+
+Projects are expected to participate in good faith in the escalation process. 
+
+## Appeals
+
+Victim(s) and offender(s) of an incident have thirty (30) days to appeal to the CoC Team the decision made in response to an incident report if they believe the process was not properly followed.
+
+If the initial report was already managed by the CoC Team, a different set of CoC Team members should be chosen to manage the appeal process. The CoC Team may choose to involve external people if needed.
+
+If the appeal follows an incident report managed by a project, the project is expected to participate in the appeal process in good faith. 
+
+Appeals are not suspensive.
+
+[CoC]: https://code-of-conduct.openjsf.org/
+[Charter]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md
+[CoC Team Charter]: https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/COC_TEAM_CHARTER.md
```

Comment:
This file does not exist

Reply:
🤦 Thanks for catching this.

Your broader point that this file needs to actually contain content to merge the PR is well taken and I agree.

Reply:
Resolving this as I've added a task to complete the CoC Team charter before merging this PR.